src/utils/api-service.ts

```diff
@@ -0,0 +1,213 @@
+    })
+  },
+}
+export const navigationAPIService = {
+  // Navigation
+  getTopNavigationMediakiwiAPI(request: GetNavigationRequestModel, url: string): Promise<GetNavigationResponseModel> {
+    const config = {
+      params: request,
+      withCredentials: true,
+      headers: { "original-url": url } 
+    };
+    return new Promise((resolve, reject) => {
+      axiosInstance.get<GetNavigationResponseModel>("navigation/GetTopnavigation", config)
+      .then((response) => {
+        if (response.status === serverCodes.OK) {
+          resolve(response.data);
+        }
+      })
+      .catch((err) => {
+
+        reject(err)
+      })
+    });
+  },
+  getSideNavigationMediakiwiAPI(request: GetNavigationRequestModel, url: string): Promise<GetNavigationResponseModel> {
+    const config = {
+      params: request,
+      withCredentials: true,
+      headers: { "original-url": url }
+    };
+    return new Promise((resolve, reject) => {
+      axiosInstance.get<GetNavigationResponseModel>("navigation/GetSidenavigation", config)
+      .then((response) => {
+        if (response.status === serverCodes.OK) {
+          resolve(response.data);
+        }
+      })
+      .catch((err) => {
+        reject(err);
+      })
+    });
+  },
+  getSitesMediakiwiAPI(request: GetNavigationRequestModel, url: string): Promise<GetSitesResponseModel> {
+    const config = {
+      params: request,
+      withCredentials: true,
+      headers: { "original-url": url }
+    };
+    return new Promise((resolve, reject) => {
+      axiosInstance.get<GetSitesResponseModel>("navigation/GetSites", config)
+      .then((response) => {
+        if (response.status === serverCodes.OK) {
+          resolve(response.data);
+        }
+      })
+      .catch((err) => {
+
+        reject(err);
+      })
+    });
+  },
+}
+export const contentAPIService = {
+  // Content 
+  getContentMediakiwiAPI(request: GetContentMediakiwiRequestModel, url: string) : Promise<GetContentMediakiwiResponseModel> {
+    const config = {
+      params: request,
+      withCredentials: true,
+      headers: { "original-url": url }
+    };
+    return new Promise((resolve, reject) => {
+      axiosInstance.get<GetContentMediakiwiResponseModel>("content/GetContent", config)
+      .then((response) => {
+        if (response.status === serverCodes.OK) {
+          resolve(response.data);
+        } 
+      })
+      .catch((err) => {        
+        store.dispatch(UITypes.SET_NOTIFICATION, { message: "Something went wrong", actionType: NotificationActionTypes.ALERT, actionText: "OK" });
+        reject(err);
+      })
+    });
+  },
+  postContentMediakiwiAPI(request: PostContentMediakiwiRequestModel, url: string): Promise<PostContentMediakiwiResponseModel> {
+    const config = {
+      withCredentials: true,
+      headers: { 
+        "original-url": url,
+      }
+    };
+    return new Promise((resolve, reject) => {
+      axiosInstance.post<any>("content/PostContent", request, config)
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8560396)
```suggestion
      axiosInstance.post<unknown>("content/PostContent", request, config)
```